cernopendata/templates/cernopendata_records_ui/records/detail.html

```diff
@@ -115,7 +115,7 @@ <h2>Disclaimer</h2>
 {% block javascript %}
 {{ super() }}
 
-<script>
+<!-- <script>
```

Comment:
Unless @mvidalgarcia doesn't need this for his current work on the files box, perhaps you can delete this code straightaway?

Reply:
I will need it as there's important logic to base my work on. I can get rid of it when the new logic is implemented.